tastyworks/models/option_chain.py

```diff
@@ -46,12 +47,33 @@ def get_all_strikes(self):
         return self._get_filter_strategy('strike')
 
     def get_all_expirations(self):
-        return self._get_filter_strategy('expiry')
+        return self._get_filter_strategy('expiration_date')
 
 
 async def get_option_chain(session, underlying: Underlying, expiration: date = None) -> OptionChain:
-    LOGGER.debug('Getting options chain for ticker: %s', underlying.ticker)
+    LOGGER.info('Getting options chain for ticker: %s', underlying.ticker)
```

Comment:
What is the reason for changing this to `info` level? Will it not make the API too verbose?

Reply:
Yep sorry I just was not sure how to see the debug information in the terminal when I was running this.